lib/plain/endpoints/usage.ts

```diff
@@ -0,0 +1,30 @@
+import { AxiosInstance } from 'axios'
+import * as raw from './raw'
+import { UsageProps } from '../../entities/usage'
+import { CollectionProp, QueryParams } from './common-types'
+
+export const getSpaceUsage = (
+  http: AxiosInstance,
+  params: { organizationId: string } & QueryParams
```

Comment:
What about a `GetUsageParams = GetOrganizationParams & QueryParams`?

Reply:
I'm actually not sure if query params are exactly the same for these endpoints but that's that we have now. I'd rather keep them inline if it's used only in 2 places.

Reply:
As we talked ages ago I always prefer one single type for the params so that they can be reused on the consumer side without inferring the function signature or the need to duplicate that intersection. But lets stick with it for time being.